VTUL/iawa
# Update VT logo and site colors to conform to VT Brand Guide

**Update VT logo and site colors to conform to VT Brand Guide.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1690) (:star:)

# What does this Pull Request do? (:star:)
Update VT logo and site colors to conform to VT Brand Guide

# What's the changes? (:star:)
* Replace old "pylons" logo with the new official VT logo
* Update the values for maroon, orange, etc and add some new colors to vtul/colors.css.scss to reflect the new university styles https://vt.edu/content/vt_edu/en/brand-center/index.html

# How should this be tested?
* Check to make sure that new university logo and updated colors are rendered in both dev and production rails modes
* Should look like this:
<img width="1380" alt="screen shot 2019-01-29 at 1 42 31 pm" src="https://user-images.githubusercontent.com/1202435/51932270-d1b34c80-23cc-11e9-8ee4-964d891cdc3e.png">


# Additional Notes:
* Branch: `LIBTD-1690`

# Interested parties
@shabububu 

(:star:) Required fields
